app/assets/stylesheets/views/_topics.scss

```diff
@@ -123,7 +123,7 @@
   .browse-container {
     background-color: $page-colour;
     min-height: 500px;
-    padding: 0 30px 45px 30px;
+    padding: 0 0 45px 0;
```

Comment:
I agree with @carolinegreen, as long as the zeroes for all those values need to be explicitly set. See also: https://github.com/alphagov/govuk-lint/commit/dcdd0a97307fd8f982c9bb5ccbdc1d11ee193526